app/scripts/models/form-prefill.js

```diff
@@ -18,7 +18,8 @@ define([
     defaults: {
       email: null,
       password: null,
-      year: null
+      year: null,
+      age: null
```

Comment:
As an aside, I kinda feel like adding a list of defaults here breaks encapsulation with the views. The form-prefill model shouldn't really care what's stored in it. Bad decision on my part?